builtins/amp-img.js

```diff
@@ -85,7 +87,11 @@ export function installImg(win) {
       }
       this.img_.setAttribute('src', src);
 
-      return loadPromise(this.img_);
+      return loadPromise(this.img_).catch(error => {
+        if (isProxyOrigin(src)) {
```

Comment:
I purposefully added a test with the relative url - and it seems to work, it uses parseUrl function which i think takes care of this ?